index.js

```diff
@@ -18,6 +18,12 @@ module.exports = function(env, callback) {
       nenv.addFilter(name, filter);
     });
   }
+  if(env.config.nunjucks && env.config.nunjucks.filtermodules) {
+    env.config.nunjucks.filtermodules.map( function (name) {
+      var filter = require(name);
+      filter.install(nenv);
```

Comment:
Hmmm yea. It seems to be kind of a convention… maybe.
I've seen it in [nunjucks-eval](https://github.com/alexandrevicenzi/nunjucks-eval/blob/master/index.js#L22), [nunjucks-date](https://github.com/techmsi/nunjucks-date/blob/master/index.js#L40) but not in [nunjucks-autoescape](https://github.com/SamyPesse/nunjucks-autoescape/blob/master/index.js) or [nunjucks-mongoose](https://github.com/metasansana/nunjucks-mongoose/blob/master/index.js) so maybe we need a more flexible way of loading such modules.

Maybe it's even possible with todays set of configuration options but I don't see how that would work without writing another .js file.